### PR TITLE
Add two new badooga homebrews

### PR DESCRIPTION
**Pull request by badooga**
Adds `badooga; Daedric Imports.json` and `badooga; Minecraft 5e.json`

### CODE REVIEW

File: creature/badooga; Daedric Imports.json [hunk 1]
```diff
@@ -0,0 +1,2103 @@
...
+				"formula": "28d8 + 224",
+				"average": 350
+			},
+			"speed": {
+				"walk": 40,
+				"fly": {
+					"number": 40,
+					"condition": "(Dragon Aspect)"
+				},
+				"canHover": false
```
Comment (TheGiddyLimit) [inline creature/badooga; Daedric Imports.json:1674]:
```suggestion
```

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
my bad; the schema got updates, but the builder can still output "bad" values. Fixed for next 5etools release

File: creature/badooga; Daedric Imports.json [hunk 2]
```diff
@@ -0,0 +1,2102 @@
...
+			"hp": {
+				"formula": "28d8 + 224",
+				"average": 350
+			},
+			"speed": {
+				"walk": 40,
+				"fly": {
+					"number": 40,
+					"condition": "(Dragon Aspect)"
+				},
```
Comment (TheGiddyLimit) [inline creature/badooga; Daedric Imports.json:1673]:
```suggestion
				}
```

File: creature/badooga; Minecraft 5e.json [hunk 3]
```diff
@@ -0,0 +1,1512 @@
+{
+	"siteVersion": "1.171.0",
```
Comment (revilowaldow) [inline creature/badooga; Minecraft 5e.json:2]:
```suggestion

```

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**